src/shared/components/create-case-filters/create-case-filters.component.ts

```diff
@@ -1,30 +1,30 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, Input, OnChanges, Output, EventEmitter, SimpleChanges } from '@angular/core';
 import { FormControl, FormGroup } from '@angular/forms';
-import { Router } from '@angular/router';
-import { Subject } from 'rxjs/Subject';
 import { Jurisdiction } from '../../domain/definition/jurisdiction.model';
 import { CaseTypeLite } from '../../domain/definition/case-type-lite.model';
 import { CaseEvent } from '../../domain/definition/case-event.model';
-import { HttpError } from '../../domain/http/http-error.model';
-import { DefinitionsService } from '../../services/definitions/definitions.service';
 import { OrderService } from '../../services/order/order.service';
-import { AlertService } from '../../services/alert/alert.service';
-import { CREATE_ACCESS } from '../../domain/case-view/access-types.model';
-import { CallbackErrorsContext } from '../error/domain/error-context';
+import { CreateCaseFiltersSelection } from './create-case-filters-selection.model';
+import { HttpError } from '../../domain';
 
 @Component({
   selector: 'ccd-create-case-filters',
   templateUrl: './create-case-filters.html'
 })
-export class CreateCaseFiltersComponent implements OnInit {
-  static readonly TRIGGER_TEXT_START = 'Start';
-  static readonly TRIGGER_TEXT_CONTINUE = 'Ignore Warning and Start';
+export class CreateCaseFiltersComponent implements OnChanges {
 
   @Input()
-  formGroup: FormGroup = new FormGroup({});
-
   jurisdictions: Jurisdiction[];
-  callbackErrorsSubject: Subject<any> = new Subject();
+  @Input()
```

Comment:
Why are we passing error as an input remind me again?

Reply:
Because errors are triggered by the consumer.

Reply:
... and they affect the component state.

Reply:
👍